apps/docs/src/app/core/component-docs/popover/examples/popover-complex-example/popover-complex-example.component.html

```diff
@@ -17,16 +17,16 @@
             <button fd-button label="With Header"></button>
         </fd-popover-control>
         <fd-popover-body>
-            <div fd-popover-body-header>
-                <div fd-bar [cozy]="true">
+			<div fd-popover-body-header>
+                <div fd-bar [cozy]="true" tabindex="0">
                     <div fd-bar-left>
                         <fd-bar-element>Cozy Header</fd-bar-element>
                     </div>
                 </div>
             </div>
-            <ul fd-list>
-                <li fd-list-item *ngFor="let option of list">
-                    <a [attr.href]="option.url" fd-list-title>{{ option.text }}</a>
+            <ul fd-list tabindex="-1">
+                <li fd-list-item *ngFor="let option of list" tabindex="0">
```

Comment:
Arrow keys are already working correctly here on main branch, no need for these changes